### ISSUE TITLE
FileNotFoundError in Yin-Yang Software: Missing Konsole Profile Leads to Configuration Failure

### ISSUE THREAD

**Issue by bphd** [oskarsh/Yin-Yang#211]
encounters a FileNotFoundError while executing the Yin-Yang software on an Arch Linux system. The error message points to the absence of a Konsole profile file, which prevents the software's configuration process from completing successfully.
 
 The Yin-Yang software relies on a specific Konsole profile file located at '/home/$USER/.local/share/konsole/Konsole.profile' to complete its configuration. However, this file is missing from the system, resulting in a FileNotFoundError.
 
 The software should Create the Konsole profile file, Populate the Konsole profile
 
 ```shell
 % killall koi yin-yang; koi & yin-yang &                                                                                                     ~ archlinux
yin-yang: no process found
[2] 1930612
[3] 1930613
 %                                                                                                                                            ~ archlinux
[1]    1930529 terminated  koi
QSocketNotifier: Can only be used with threads started with QThread
 % qt.core.plugin.loader: /usr/lib/qt6/plugins/position/libqtposition_geoclue2.so: cannot open: Permission denied                             ~ archlinux
nmea: No known GPS device found.
Traceback (most recent call last):
  File "/opt/yin-yang/main.py", line 13, in <module>
    from src import daemon_handler
  File "/opt/yin-yang/src/daemon_handler.py", line 7, in <module>
    from src.config import ConfigWatcher, config
  File "/opt/yin-yang/src/config.py", line 474, in <module>
    config = ConfigManager()
             ^^^^^^^^^^^^^^^
  File "/opt/yin-yang/src/config.py", line 179, in __init__
    self.load()
  File "/opt/yin-yang/src/config.py", line 246, in load
    pl.theme_light = config_loaded['plugins'][pl.name.lower()]['light_theme']
    ^^^^^^^^^^^^^^
  File "/opt/yin-yang/src/plugins/konsole.py", line 42, in theme_light
    self.update_profile(False, value)
  File "/opt/yin-yang/src/plugins/konsole.py", line 174, in update_profile
    self.create_profiles()
  File "/opt/yin-yang/src/plugins/konsole.py", line 189, in create_profiles
    copyfile(self.user_path / self.default_profile, light_profile)
  File "/usr/lib/python3.11/shutil.py", line 256, in copyfile
    with open(src, 'rb') as fsrc:
         ^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/home/$USER/.local/share/konsole/Konsole.profile'

[3]  + 1930613 exit 1     yin-yang
 %                                                                                                                                            ~ archlinux
```

### Enabled plugins

What plugins did you have enabled? If not relevant, you can remove this.

- [ ] Atom
- [ ] Brave
- [ ] Colors
- [ ] Custom Script
- [ ] Firefox
- [ ] Gedit
- [ ] GTK
- [ ] Konsole
- [ ] Kvantum
- [ ] Only Office
- [ ] System
- [ ] VSCode
- [ ] Wallpaper

### Affected versions

- Yin-Yang version:

- Relevant application version:
- Python version:
- Qt version:

**Comment by l0drex** [comment]
Duplicate of #200

**Comment by bphd** [comment]
Duplicated is not resolved. There is only a workaround in it

**Comment by bphd** [comment]
Still not resolved in 3.2.4.r9.g42ce4da-1
